code/game/gamemodes/events/biomass.dm

```diff
@@ -205,3 +205,34 @@
 		new/obj/effect/biomass_controller(Floor) // spawn a controller at floor
 		log_admin("Event: Biomass spawned at [Floor.loc] ([Floor.x],[Floor.y],[Floor.z]).")
 		message_admins("<span class='notice'>Event: Biomass spawned at [Floor.loc] <A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[Floor.x];Y=[Floor.y];Z=[Floor.z]'>(JMP)</a></span>")
+
+//biomass tator item, code "borrowed" from powercreeper_packet.dm
```

Comment:
instead why not just make them one in the same as some sort of deployable packet item? the only difference is the icon and name, seems a perfect time for inheritance

Reply:
I tried implementing it as a child of the powercreeper packet, but it didn't want to spawn biomass. There's also a small difference in the examine text (the biomass one has some flavour too). I can try to rework it a bit however.

Reply:
```
/obj/item/deployable_packet/
	desc = ""
	w_class = W_CLASS_TINY
	var/activated = 0
	var/deployeditem = null

/obj/item/deployable_packet/attack_self(mob/user)
	if(!istype(user))
		return
	if(activated)
		return

	to_chat(user, "<span class='warning'>You shake \the [src].</span>")
	to_chat(user, "<span class='danger'>It starts moving weirdly!</span>")
	activated = 1

	spawn(3 SECONDS)
		new deployeditem(get_turf(src))
		qdel(src)

/obj/item/deployable_packet/biomass
	name = "Biomass Packet"
	icon = 'icons/obj/biomass.dmi'
	icon_state = "packet"
	deployeditem = /obj/effect/biomass_controller

/obj/item/deployable_packet/powercreeper/examine(mob/user, size, show_name)
	. = ..()
	to_chat(user, "Derek Baum VII's latest innovation. Carries a cryo-stabilized sample of biomass tissue.")
	to_chat(user, "It reads:\nStep 1: Shake to active.\nStep 2: Throw.\nStep 3: Enjoy.")
	if(activated)
		to_chat(user, "<span class='danger'>It's bubbling weirdly!</span>")
			
/obj/item/deployable_packet/powercreeper
	name = "powercreeper Packet"
	icon = 'icons/obj/structures/powercreeper.dmi'
	icon_state = "packet"
	deployeditem = /obj/structure/cable/powercreeper
	
/obj/item/deployable_packet/powercreeper/examine(mob/user, size, show_name)
	. = ..()
	to_chat(user, "Derek Baum VII's favourite creation. Carries a self-replicating sample of powercreeper.")
	to_chat(user, "It reads:\nStep 1: Shake to active.\nStep 2: You have 3 seconds to run.")
	if(activated)
		to_chat(user, "<span class='danger'>It's sizzling weirdly!</span>")

```

Something like this?

Reply:
Yes, that would work, although you made a typo in the path of the examine for biomass which would override powercreeper's examine

Reply:
Alright, I'll implement it now then (and thanks for noticing that typo).

Reply:
Would it be better to have it as its own file (deployable packet.dm)?

Reply:
since powercreeper_packet was its own file I would delete that one and make a new one named as you say

Reply:
Alright, I think that should be it. Please check.

Reply:
I had a brainblast for space vine packets, but that's for another PR.

Reply:
@Kurfursten